easybuild/easyconfigs/c/cuDNN/cuDNN-7.5.0.56-CUDA-10.0.130.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = "7.5.0.56"
+cuda_version = '10.0.130'
+
+cuda_version_major_minor = '.'.join(cuda_version.split('.')[:2])
+
+versionsuffix = '-CUDA-%s' % cuda_version
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+# Download as cudnn-10.0-linux-x64-v7.5.0.56.tgz, rename to cudnn-10.0-linux-x64-v<VERSION>.tgz
+#source_urls = ['https://developer.nvidia.com/compute/machine-learning/cudnn/secure/v7.5.0.56/prod/10.0_20190219/']
```

Comment:
@henkela You'll need to add a space after `#` to make the style check happy... Or maybe just drop this line as well?